static/src/javascripts/lib/cookies.js

```diff
@@ -1,4 +1,12 @@
 // @flow
+import reportError from 'lib/report-error';
+
+const ERR_INVALID_COOKIE_NAME = `Cookie must not contain invalid characters (space, tab and the following characters: '()<>@,;"/[]?={}')`;
+
+// subset of https://github.com/guzzle/guzzle/pull/1131
+const isValidCookieValue = (...names: string[]): boolean =>
```

Comment:
Might be worth renaming this given it accepts multiple args? Although I think it would be even clearer to not make this function multi-arg in the first place. Reason being multi-arg immediately complicates error reporting (which name was invalid?) and we have things like some/map etc when the consumer has multiple values.